clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -28,6 +30,7 @@
  *
  */
 final class ClusterConnectionStates {
+    private static final Logger log = LoggerFactory.getLogger(ClusterConnectionStates.class);
```

Comment:
Is there a specific reason for adding this in? I can't find any usages of the logger ..